.github/workflows/ci.yml

```diff
@@ -49,6 +49,9 @@ jobs:
           - name: pytest
             cmd: pytest --cov=pymodbus --cov=test --cov-report=term-missing --cov-report=xml -v --full-trace --timeout=20
             type: test
+          - name: mypy
```

Comment:
Please move this up to black etc, so we group all linters together, and tests together.